### PR TITLE
Clean up error return on kernel selection

### PR DESCRIPTION
**Pull request by aljames-a12**
The user should only need to know that their string does not constitute a valid number, encapsulate their unparsed choice in quotes in error messages, submit it to Atoi with spaces stripped of the left and right.

### CODE REVIEW

File: pkg/boot/menu/menu.go [hunk 1]
```diff
@@ -51,12 +52,9 @@ type Entry interface {
...
 func parseBootNum(choice string, entries []Entry) (int, error) {
-	num, err := strconv.Atoi(choice)
-	if err != nil {
-		return -1, fmt.Errorf("%s is not a valid entry number: %v", choice, err)
-	}
-	if num < 1 || num > len(entries) {
-		return -1, fmt.Errorf("%s is not a valid entry number", choice)
+	num, err := strconv.Atoi(strings.TrimSpace(choice))
+	if err != nil || num < 1 || num > len(entries) {
+		return -1, fmt.Errorf("'%s' is not a valid entry number", choice)
```
Comment (rjoleary) [inline pkg/boot/menu/menu.go:57]: Instead of `'%s'` use `%q`

Comment (aljames-a12) [reply pkg/boot/menu/menu.go]: Done.

Comment (aljames-a12) [reply pkg/boot/menu/menu.go]: Actually is %q valid?

%q	a single-quoted character literal safely escaped with Go syntax.

choice isn't necessarily a single character. The purpose of the ' ' was to encapsulate the entire string in a displayable format, with spaces/tabs/etc included, would it be better to escape some "s?

Comment (aljames-a12) [reply pkg/boot/menu/menu.go]: Ah, never-mind, I was misinterpreting 'character literal' to be a single character, it is not

**Review by rjoleary — DISMISSED** [review]

**Comment by rjoleary** [comment]
Go to this page: https://app.circleci.com/pipelines/github/aljames-a12/u-root. Press "Project Settings" (in the top left), then "Stop Building" (the red button).

For some reason, CircleCI associated your PR with the wrong organization.

**Review by rjoleary — APPROVED** [review]

**Pull request merged**